oak-auth-external/src/main/java/org/apache/jackrabbit/oak/spi/security/authentication/external/impl/DefaultSyncConfigImpl.java

```diff
@@ -272,6 +272,23 @@ public class DefaultSyncConfigImpl extends DefaultSyncConfig {
     )
     public static final String PARAM_GROUP_PATH_PREFIX = "group.pathPrefix";
 
+    /**
+     * @see DefaultSyncConfig.Group#getDynamicGroups()
+     */
+    public static final boolean PARAM_GROUP_DYNAMIC_GROUPS_DEFAULT = false;
+    
+    /**
+     * @see DefaultSyncConfig.Group#getDynamicGroups()
+     */
+    @Property(
+            label = "Group 'Dynamic Groups'",
```

Comment:
This label seems difficult to understand.

It comes to my mind "Dynamic Group Membership" since the group membership gets dynamically calculated. Or maybe "Dynamically Resolved Group Membership" if we want to be more precise?

Reply:
agree.... dynamic membership is already used but i will think about a better name.